feat: updates to ban process (#67)
electron/governance: playbooks/responses/how-it-works.md

> Open source is a significantly volunteer-driven and collaborative system of production. Because of this structure, time and energy to get things done is the scarcest of resources in open source projects. We want Electron to be the best that it can be, but we also have to be realistic about what the team maintaining Electron can achieve. This means that:

Reviewer: ```suggestion
Open source is largely volunteer-driven and collaborative. Because of this, time and energy to get things done are often the scarcest resources. We want Electron to be the best that it can be, but we also have to be realistic about what the team maintaining Electron can achieve. This means that:
```

Minor edits for the PR's goal of making the warning shorter

> * We may not give you an ETA for when an issue will get investigated or fixed

Reviewer: ```suggestion
* We might not have an ETA for when an issue will get investigated or fixed.
```

Softens the tone a little. I'm a little torn on these bullet point changes -- this softer voice is the one I'd personally use, but I'm not sure it's congruent with the message's warning nature. So these three are more of a for-your-consideration

> * We may decide that we aren't going to implement something you want implemented

Reviewer: ```suggestion
* Something you want implemented might not get implemented.
```

> * We may decide that we aren't going to accept a change that you want merged

Reviewer: ```suggestion
* A change you want merged might not get merged.
```
